libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -149,6 +165,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Description: Controls how often the GPS should provide a position update. Lowering below 5Hz is not allowed
     // @Units: milliseconds
     // @Values: 100:10Hz,125:8Hz,200:5Hz
+    // @Range: 100 200
```

Comment:
Yeah there are, Septentrio is capable of 20Hz updates, and there are some other units on the market that do it as well. Now the relative value of these positions is a bit up for debate, but they are definitely out there and can be acquired now :)